presto-kafka/src/main/java/com/facebook/presto/kafka/KafkaSplit.java

```diff
@@ -42,7 +42,7 @@
     private final int partitionId;
```

Comment:
nit: we should also update the javadoc of this class to keep it in sync with the implementation as now we only read from the leader.

>> a partition can be processed by reading segment files off different Kafka nodes (in case of replication)